app/renderer/src/components/Inspector/Inspector.module.css

```diff
@@ -73,6 +73,10 @@
   height: 100%;
 }
 
+.inspector-main .interaction-tab-container :global(.ant-input-affix-wrapper) {
```

Comment:
This does not look good at the minimum window width:
![image](https://github.com/appium/appium-inspector/assets/37242620/6cc48f74-23f8-459a-a977-de40cc4718d8)
Although I'm not sure how to best scale this. Ideally it should shrink without affecting the card title, but the actual behavior is the opposite.

Reply:
Also I would suggest defining a class for the searchbar if you intend to change its CSS. This particular line is rather long and it's not obvious what it's meant to affect.

Reply:
I just tweaked a bit to make it better for smaller screens. Do let me know if it looks good.

<img width="320" alt="Screenshot 2024-08-19 at 5 38 49 PM" src="https://github.com/user-attachments/assets/27420cba-16af-45d5-899b-c11f53273aa3">

Reply:
It is better in that there's no linebreak, but ideally the search bar and icons should not hide the 'App Source' title - instead the title should hide the search bar/buttons. Not sure how possible this is.